lib/data-serializer.js

```diff
@@ -1,22 +1,27 @@
 'use strict'
 
+const Poller = require('./poller')
 const SplitioApiBinding = require('./splitio-api-binding')
 
 class DataSerializer {
   constructor ({
     splitioApiKey,
     pollingRateSeconds,
+    poller,
     serializeSegments
   }) {
-    this._splitioApiBinding = new SplitioApiBinding({ splitioApiKey })
-    this.pollingRateSeconds = pollingRateSeconds || 300
-    this.serializeSegments = serializeSegments || false
+    this.poller = poller || new Poller({
+      pollingRateSeconds,
+      serializeSegments,
+      splitioApi: new SplitioApiBinding({ splitioApiKey })
```

Comment:
I like the term 'binding' to reduce confusion. can we keep the naming consistent and make this variable splitioApiBinding?